projects/graph/src/draw.py

```diff
@@ -1,14 +1,69 @@
 """
```

Comment:
Consider adding a `shebang` to the top of your Python files. They help to ensure the correct version is executing the file and in some cases they make them easier to work with by making them executable without having to type *python3* in front of it every time in the terminal.